joomla/joomla-cms
# Formatted the calendar widget

Pull Request for Issue #36933  .

### Summary of Changes
solved issue number 3/3.
Stylized the time drop-down menu according to bootstrap 4 and centred the calendar table in the widget for the calendar to look more symmetrical


### Testing Instructions
Open the calendar widget in any of the tabs you wish to, notice the change in the buttons from the previous versions


### Actual result BEFORE applying this Pull Request
calendar looked like this
<img width="421" alt="Screenshot 2022-02-10 at 7 13 25 PM" src="https://user-images.githubusercontent.com/78909027/153420086-413085ed-34f4-4001-82cd-ebcc65e52811.png">


### Expected result AFTER applying this Pull Request
it should look like this
<img width="700" alt="Screenshot 2022-02-10 at 7 13 48 PM" src="https://user-images.githubusercontent.com/78909027/153420147-001ae3a9-cf59-424b-8d58-e67c179fe6a0.png">



### Documentation Changes Required
No changes required
